### ISSUE TITLE
Add newline between sfc level tags: script, template, style

### ISSUE THREAD

**Issue by zhaojjiang** [vuejs/eslint-plugin-vue#2719]
**Please describe what the rule should do:**

**What category should the rule belong to?**

[X] Enforces code style (layout)
[ ] Warns about a potential error (problem)
[ ] Suggests an alternate way of doing something (suggestion)
[ ] Other (please specify:)

**Provide 2-3 code examples that this rule should warn about:**

```vue
<script>
</script>
<template>
</template>
<style>
</style>
```

--format to-->

```vue
<script>
</script>

<template>
</template>

<style>
</style>
```

**Additional context**

**Comment by FloEdelmann** [comment]
Already possible: https://eslint.vuejs.org/rules/padding-line-between-blocks.html

**Comment by zhaojjiang** [comment]
thanks for your reply, I didn't notice this option.
